internal/pkg/workspace/workspace.go

```diff
@@ -125,6 +125,21 @@ func (ws *Service) writeSummary(projectName string) error {
 	return ws.fsUtils.WriteFile(summaryPath, serializedWorkspaceSummary, 0644)
 }
 
+//LocalApps returns the name of all the local apps
```

Comment:
nit: can you put a space after the slashes, and end the comment with a dot: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

> Comments should begin with the name of the thing being described and end in a period